src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -249,11 +255,20 @@ public override async Task<DataTable> ExecuteSqlQueryAsync(string cloudServiceNa
             }
             catch (HttpRequestException)
             {
-                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider",
-                    $"message:Observer SQL query request failed, query:{query}, statusCode:{response.StatusCode}, response:{result}");
+                var logMessage = $"message:Observer SQL query request failed, route:{route}, query:{query}, " +
+                    $"statusCode:{response.StatusCode}, response:{result}, latencyMs:{timer.ElapsedMilliseconds}";
+                Logger.LogDataProviderMessage(RequestId, "ObserverDataProvider", logMessage);
                 throw;
             }
 
+            if (timer.ElapsedMilliseconds > 1000)
```

Comment:
Is this logging for slow requests only when exception happened?

Reply:
Lets log this in a finally clause. We can log both success and failure SQL query calls. This way we can also look back to the SQL query that people are using for even successful but slow calls.